mapnik_palette compile error on 0.7.x branch

I'm getting a compile error on the 0.7.x branch

```
mapnik-config --git-describe v2.2.0
node -v v0.10.12
```

```
../src/mapnik_palette.cpp:90:27: error: variable ‘std::ostringstream str’ has initialiser but incomplete type
```

full log:

```
andy@eiger:~/temp/node-mapnik$ npm install
npm http GET https://registry.npmjs.org/jshint
npm http GET https://registry.npmjs.org/mocha
npm http 304 https://registry.npmjs.org/mocha
npm http 304 https://registry.npmjs.org/jshint
npm WARN package.json jshint@0.5.9 No repository field.
npm http GET https://registry.npmjs.org/argsparser
npm http GET https://registry.npmjs.org/minimatch
npm http 304 https://registry.npmjs.org/minimatch
npm http GET https://registry.npmjs.org/jade/0.26.3
npm http GET https://registry.npmjs.org/commander/2.0.0
npm http GET https://registry.npmjs.org/growl
npm http GET https://registry.npmjs.org/diff/1.0.7
npm http GET https://registry.npmjs.org/debug
npm http GET https://registry.npmjs.org/mkdirp/0.3.5
npm http GET https://registry.npmjs.org/glob/3.2.3
npm http 304 https://registry.npmjs.org/argsparser
npm http 304 https://registry.npmjs.org/commander/2.0.0
npm http 304 https://registry.npmjs.org/jade/0.26.3
npm http 304 https://registry.npmjs.org/diff/1.0.7
npm http 304 https://registry.npmjs.org/growl
npm WARN package.json jade@0.26.3 No readme data.
npm WARN package.json growl@1.7.0 No repository field.
npm http GET https://registry.npmjs.org/sigmund
npm http GET https://registry.npmjs.org/lru-cache
npm http 304 https://registry.npmjs.org/debug
npm http 304 https://registry.npmjs.org/mkdirp/0.3.5
npm http 304 https://registry.npmjs.org/glob/3.2.3
npm http 304 https://registry.npmjs.org/sigmund
npm http 304 https://registry.npmjs.org/lru-cache
npm http GET https://registry.npmjs.org/graceful-fs
npm http GET https://registry.npmjs.org/inherits
npm http GET https://registry.npmjs.org/commander/0.6.1
npm http GET https://registry.npmjs.org/mkdirp/0.3.0
npm http 304 https://registry.npmjs.org/graceful-fs
npm http 304 https://registry.npmjs.org/inherits
npm http 304 https://registry.npmjs.org/commander/0.6.1
npm http 304 https://registry.npmjs.org/mkdirp/0.3.0

> mapnik@0.7.23 install /home/andy/temp/node-mapnik
> node-gyp rebuild

make: Entering directory `/home/andy/temp/node-mapnik/build'
  CXX(target) Release/obj.target/_mapnik/src/node_mapnik.o
  CXX(target) Release/obj.target/_mapnik/src/mapnik_map.o
  CXX(target) Release/obj.target/_mapnik/src/mapnik_color.o
  CXX(target) Release/obj.target/_mapnik/src/mapnik_geometry.o
  CXX(target) Release/obj.target/_mapnik/src/mapnik_feature.o
  CXX(target) Release/obj.target/_mapnik/src/mapnik_image.o
  CXX(target) Release/obj.target/_mapnik/src/mapnik_image_view.o
  CXX(target) Release/obj.target/_mapnik/src/mapnik_grid.o
  CXX(target) Release/obj.target/_mapnik/src/mapnik_grid_view.o
  CXX(target) Release/obj.target/_mapnik/src/mapnik_js_datasource.o
  CXX(target) Release/obj.target/_mapnik/src/mapnik_memory_datasource.o
  CXX(target) Release/obj.target/_mapnik/src/mapnik_palette.o
../src/mapnik_palette.cpp: In static member function ‘static v8::Handle<v8::Value> Palette::ToString(const v8::Arguments&)’:
../src/mapnik_palette.cpp:90:27: error: variable ‘std::ostringstream str’ has initialiser but incomplete type
../src/mapnik_palette.cpp:107:1: warning: control reaches end of non-void function [-Wreturn-type]
make: *** [Release/obj.target/_mapnik/src/mapnik_palette.o] Error 1
make: Leaving directory `/home/andy/temp/node-mapnik/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.EventEmitter.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:789:12)
gyp ERR! System Linux 3.2.0-58-generic
gyp ERR! command "node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/andy/temp/node-mapnik
gyp ERR! node -v v0.10.12
gyp ERR! node-gyp -v v0.10.0
gyp ERR! not ok 
npm ERR! weird error 1
npm ERR! not ok code 0
```

--- a/src/mapnik_palette.cpp
+++ b/src/mapnik_palette.cpp
@@ -11,6 +11,7 @@
 // stl
 #include <vector>
 #include <iostream>
+#include <sstream>
 #include <iomanip>
 
 Persistent<FunctionTemplate> Palette::constructor;

